src/Modules/OrchardCore.Commerce/Recipes/OrchardCore.Commerce.Development.Tests.Setup.recipe.json

```diff
@@ -19,6 +19,10 @@
         {
           "executionid": "OrchardCore.Commerce",
           "name": "OrchardCore.Commerce.Development.Setup"
+        },
+        {
+          "executionid": "OrchardCore.Commerce",
+          "name": "TestProduct"
```

Comment:
The `PriceVariantsProduct` was OK for testing, but it didn't work, since it didn't have `ProductAttributeField`. I moved it to a different recipe and created a new one, that has a `ProductAttributeField`.